build/install-istio-with-kind.sh

```diff
@@ -7,7 +7,7 @@ set -x
 GOARCH=$(go env GOARCH)
 GOOS=$(go env GOOS)
 KIND_VERSION=0.9.0
-ISTIO_VERSION=1.7.0
+ISTIO_VERSION=1.8.2
```

Comment:
I think the latest minor version of the oldest supported series is a decent choice, what do you think?